### PR TITLE
Update catalan messages.po

### PR DESCRIPTION
**Pull request by ivanbea**
New lines translated
You can check it @jordimas @darccio @ferranrego @MiquelAdell @mvdan @johnnydement @Mejans @surfdude29

### CODE REVIEW

File: src/locale/locales/ca/messages.po [hunk 1]
```diff
@@ -4085,7 +4085,7 @@ msgstr "Filtres de resposta"
 #: src/view/com/posts/FeedItem.tsx:285
 msgctxt "description"
 msgid "Reply to <0><1/></0>"
-msgstr ""
+msgstr "Resposta a  <0><1/></0>"
```
Comment (surfdude29) [inline src/locale/locales/ca/messages.po:4088]: I think there's an extra space here after `a`

Comment (ivanbea) [reply src/locale/locales/ca/messages.po]: Well spotted, thanks!

**Comment by surfdude29** [comment]
Just to check, did you see the comment from @jordimas on your own repo?

https://github.com/ivanbea/social-app/commit/860c2c3fdd72cadb19c9489a28adb34639189b0b#r141348104

**Review by pfrazee — APPROVED** [review]
Thanks!

**Pull request merged**